Monika After Story/game/script-songs.rpy

```diff
@@ -1191,6 +1191,139 @@ label mas_song_lamour_toujours:
+    m 3esc "While some people can go their entire life listening to the same musics, not everybody's like that."
+
+    if seen_event("mas_monika_plays_or"):
+        m 3hksdlb "...I hope you won't get tired of {i}my{/i} songs anytime soon [player]. Ahaha~"
+    else:
+        m 3hksdlb "...I hope you won't get tired of {i}my{/i} song anytime soon [player]. Ahaha~"
+
+    m 1esd "Another argument you could make is it being overhyped."
+    m 1rsu "As much as I like it, it's true the lyrics and chords are pretty simple; {nw}"
+    extend 3etc "what makes this song in particular so deserving of all the attention it gets?"
+    m 3eud "...Especially when so many other songs still fly completely under the radar, no matter how advanced or ambitious they are."
+    m 1lksdlc "It's tricky to evaluate though, you can't really tell the amount of work that went into something solely by the result.{w=0.3} {nw}"
+    extend 3esd "Simple and clean does not mean unpolished."
+    m 3duu "Ultimately, it all boils down to what the song makes you feel; There's no being wrong about your tastes in music."
+    m 1efc "What bothers me is when someone complains about it just because it's trendy to go against the general opinion."
+    m 3tsd "It's like disagreeing for the sake of it helps them feel like they stand out from the crowd, like they need it to stay self-confident."
+    m 2rsc "It kinda feels...{w=0.5}a bit silly, to be honest."
+    m 2rksdld "At this point you're not even judging the song anymore, you're just trying to ascertain your own identity."
+    m 2dksdlc "It's a little sad if anything, {nw}"
+    extend 7eksdlc "defining yourself by something you hate does not seem like a very healthy thing to do in the long run."
+    m 1rsc "Yeah..."
+    m 3etc "Although...{w=0.3}there might be yet another reason why this song is controversial with some people."
+    m 3esd "Some of the members of this band have been the subject of controversies in the past due to their...{w=0.5}wild behaviors."
+    m 1hksdlb "I'm not trying to judge them or anything, it's not like I ever actually sat down and talked with them."
+    m 3esc "...But from a total stranger's perspective, this raises the question of how much can one's bad actions be excused by talent or charisma."
+    
+    if persistent._mas_pm_monika_evil or persistent._mas_pm_a_hater:
+        m 3tsu "Though if you're going out with me it's not exactly like you're a stranger to the question, right? Ehehe~"
+        m 3esc "Not that I believe what I did was unjustified, passing you by was no option."
+
+        if persistent._mas_pm_cares_about_dokis:
+            m 3dkc "...Still, in hindsight, I see there are some things I could have handled more tactfully."
+            m 3lksdld "It's not easy staying on your best behavior when you're put under pressure; even more so when it's your life on the line."
```

Comment:
```suggestion
            m 3lksdld "When it's your life on the line, it's not easy staying on your best behavior when under pressure."
```

Reply:
Maybe it's just me, but the double `When` in this sentence sounds a bit akward, maybe change it the second 'when' to 'while'?
`When it's your life on the line, it's not easy staying on your best behavior while under pressure.`

Reply:
updated